mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/InfoSection.tsx

```diff
@@ -168,6 +178,7 @@ function PoolPairInfoLine ({
             price={usdValue.text}
             containerStyle={tailwind('justify-end -mt-0.5')}
             testId={usdValue.testID}
+            lightTextStyle={isTitle === true ? tailwind('text-black') : undefined}
```

Comment:
Our tailwind uses class-names for conditional statements. You can do something like this:

```suggestion
            lightTextStyle={tailwind({ 'text-black': isTitle })}
```